packages/flutter_tools/lib/src/run_cold.dart

```diff
@@ -40,29 +38,6 @@ class ColdRunner extends ResidentRunner {
 
   @override
   Future<int> run({
-    Completer<DebugConnectionInfo> connectionInfoCompleter,
-    Completer<Null> appStartedCompleter,
-    String route,
-    bool shouldBuild: true
-  }) {
-    // Don't let uncaught errors kill the process.
-    return Chain.capture(() {
-      return _run(
-        traceStartup: traceStartup,
```

Comment:
There's a slight semantic difference here -- previously, it would use the value of `traceStartup` that was passed to its constructor if it was unspecified in `run()`.  Now it'll always default to `false`